systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -229,4 +234,20 @@ public static boolean checkLogForJSONFormat(Map<String, String> pods, String con
         }
         return isJSON;
     }
+
+    public static <T extends CustomResource & HasStatus> void logCurrentStatus(T customResource) {
```

Comment:
nice use of generics :) but please make sure when you  defining it, create some doc, where you explain, which set of types can be contains T type.